astro/src/content/docs/lifecycle/authenticate-users/single-sign-on.mdx

```diff
@@ -4,13 +4,17 @@ description: Learn how to implement single sign-on between applications using Fu
 navcategory: developer
 section: lifecycle
 subcategory: authenticate users
+codeRoot: https://raw.githubusercontent.com/FusionAuth/fusionauth-example-node-sso/main
 ---
 import Aside from 'src/components/Aside.astro';
 import InlineField from 'src/components/InlineField.astro';
+import Breadcrumb from 'src/components/Breadcrumb.astro';
+import InlineUIElement from 'src/components/InlineUIElement.astro';
 import LogoutBehaviorAllApplications from 'src/content/docs/get-started/core-concepts/_logout-behavior-all-applications.mdx';
 import SSOLogin from 'src/diagrams/docs/lifecycle/authenticate-users/sso-login.astro';
 import SSOLogout from 'src/diagrams/docs/lifecycle/authenticate-users/sso-logout.astro';
 import { YouTube } from '@astro-community/astro-embed-youtube';
+import {RemoteCode} from '@fusionauth/astro-components';
 
 This guide will walk you through setting up single sign-on (SSO) between two web applications using FusionAuth as their common authentication and authorization server. You will use the hosted login pages for your login form.
```

Comment:
Do you think it would be clearer to use ngrok to show the domain differences in the below instead of messing with etc hosts?

Reply:
Great idea to use ngrok. I'd have to figure out how to get the value of the ngrok URL to the kickstart so that it would be added to the authorized redirect URIs, but other than that it makes total sense and would be much easier for users.

Reply:
See https://github.com/FusionAuth/fusionauth-site/pull/3131#discussion_r1674807563 but we can't use ngrok (at least the free version) because it puts a page in front of the iframe request, so logout breaks anyway.